main.go

```diff
@@ -35,13 +35,18 @@ import (
 	"github.com/kuadrant/authorino/pkg/log"
 	"github.com/kuadrant/authorino/pkg/metrics"
 	"github.com/kuadrant/authorino/pkg/service"
+	"github.com/kuadrant/authorino/pkg/trace"
 	"github.com/kuadrant/authorino/pkg/utils"
+	otel_grpc "go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc"
+	otel_http "go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
+	otel_propagation "go.opentelemetry.io/otel/propagation"
```

Comment:
Do you mind separating this block? We try to keep the imports separated in at least 3 blocks:
- Go standard library packages
- `github.com/kuadrant/authorino/*` packages
- Third-party packages